src/GrumPHP/Task/PhpVersion.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+namespace GrumPHP\Task;
+
+use GrumPHP\Configuration\GrumPHP;
+use GrumPHP\Runner\TaskResult;
+use GrumPHP\Runner\TaskResultInterface;
+use GrumPHP\Task\Context\ContextInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+/**
+ * PhpVersion task
+ */
+class PhpVersion implements TaskInterface
+{
+
+    /**
+     * @var array
+     */
+    private $versions;
+
+    /**
+     * @var string
+     */
+    private $projectVersion;
+
+    /**
+     * PhpVersion constructor.
+     * @param GrumPHP $config
+     * @param array $versions
+     */
+    public function __construct(GrumPHP $config, array $versions)
+    {
+        $this->versions = $versions;
+        $options = $config->getTaskConfiguration($this->getName());
```

Comment:
You can take a look at other tasks how to set configurable options. This should be done by returning an OptionsResolver in the `getConfigurableOptions()` method. The `getConfiguration()` method used this OptionsResolver to validate the configured options.